charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.dataSource.local.hostPath .Values.dataSource.local.nodeName -}}
+data_dir={{ .Values.dataSource.local.hostPath }}
+{{- else -}}
+data_dir=$(dirname $(find /var/lib/tidb-lightning -name metadata 2>/dev/null) 2>/dev/null)
+if [ -z $data_dir ]; then
+    if [ ! -z ${FAIL_FAST} ]; then
+        exit 1
+    else
+        echo "No mydumper files are found, please exec into my container to diagnose"
+        tail -f /dev/null
+    fi
+fi
+{{ end }}
+/tidb-lightning \
+    --pd-urls={{ .Values.targetTidbCluster.name }}-pd.{{ .Release.Namespace }}:2379 \
+    --status-addr=0.0.0.0:8289 \
+    --importer={{ .Values.targetTidbCluster.name }}-importer.{{ .Release.Namespace }}:8287 \
+    --server-mode=false \
+    --tidb-user={{ .Values.targetTidbCluster.user | default "root" }} \
+    --tidb-host={{ .Values.targetTidbCluster.name }}-tidb.{{ .Release.Namespace }} \
+    --d=${data_dir} \
+    --config=/etc/tidb-lightning/tidb-lightning.toml
```

Comment:
For some old version of tidb-lightning, `tikv_gc_life_time` cannot be restored even when the procedure completed successfully. Should we handle this explicitly in the script?

Reply:
Which versions of tidb-lightning have this issue? If only too old versions, then I don't think it's worth to do it.

Reply:
Actually it has been fixed only recently by https://github.com/pingcap/tidb-lightning/pull/218 . According to the PR, it's a concurrency problem. However, I remember that I can almost always reproduce it by tidb-ansible, but I failed to do it in k8s. Maybe it's because of difference of config.